packages/vue-dot/src/functions/getFormValue/index.ts

```diff
@@ -0,0 +1,26 @@
+import { FieldValue } from './../../patterns/FormField/types.d';
+import { Form } from './../../patterns/FormBuilder/types';
+
+export interface FormValue {
+	[key: string]: FieldValue;
+}
+
+/** Return an array with field values */
+export default function getFormValue(form: Form): FormValue {
+	const data: FormValue = {};
```

Comment:
```suggestion
	const formValues: FormValues = {};
```
C'est mieux que `data` je pense !